src/drivers/Qt/ConsoleVideoConf.cpp

```diff
@@ -748,6 +748,11 @@ void ConsoleVideoConfDialog_t::winFullScreenBorderChanged(int value)
 	g_config->setOption("SDL.winFullScreenBorder", opt );
 	g_config->save();
 }
+#elif defined(WIN32)
+void ConsoleVideoConfDialog_t::winFullScreenBorderChanged(int value)
```

Comment:
If this function has no meaning for Qt 6, then it should not exist in a Qt 6 build at all.

Reply:
Removed this and fixed header.